mediamtx/entrypoint.sh

```
@@ -0,0 +1,6 @@
+#!/bin/sh
+# Substitute environment variables in the config template before starting go2rtc.
+# GO2RTC_HOST must be set to the server's public IP so WebRTC ICE candidates
+# point to a reachable address (the Docker internal IP is not browser-accessible).
+envsubst '${GO2RTC_HOST}' < /config/go2rtc.yaml.template > /config/go2rtc.yaml
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)